src/test/java/uk/gov/dhsc/htbhf/BrowserStackCapabilities.java

```diff
@@ -52,13 +56,19 @@ public static DesiredCapabilities buildDesiredCapabilities(Map<String, String> c
         DesiredCapabilities capabilities = new DesiredCapabilities();
         capabilitiesMap.forEach((key, value) -> {
             //Sadly if we pass a "true" or "false" String into here it doesn't get treated as a boolean so we must convert it.
-            Object valueToSet = ("true".equals(value) || "false".equals(value)) ? BooleanUtils.toBooleanObject(value) : value;
+            Object valueToSet = isTrueOrFalseString(value) ? BooleanUtils.toBooleanObject(value) : value;
```

Comment:
Couldn't this simply be `Boolean.valueOf(value)`? It'll be false unless the string is 'true', but I think that would be ok?

Reply:
No, you've missed the point of this, this method needs to figure out if its the String value of `true` or `false` first, if its either of those then convert it to a boolean, otherwise leave it alone, otherwise a normal String value (other than `true` or `false`) will always become false, which isn't correct. Does that make sense?

Reply:
yup, makes sense.